### DISCUSSION TITLE
Feature Idea: Manual override for keep-alive

### DISCUSSION THREAD

**Discussion by peteh** [wg-easy/wg-easy#1204] [Ideas]
Hi have some services that I want to be reachable from other clients all the time. 

However, the clients don't need to be available all the time, thus they don't really need a keep alive. 

So the keepalive setting as a global setting is not fully sufficient and it would be cool to deviate from the global setting for some clients. 

Example use case: 
One client is a homeassistant that should be reachable by all connected clients. However, I don't care that the Mobile phone i'm using to reach is keeps its connection alive. 

Thus I would go to some advanced setting for this particular client and overwrite the default (e.g. selection: default/keepalive + timeout/no keepalive)

What do you think?

**Comment by pheiduck** [comment]
This option already exists see:
https://github.com/wg-easy/wg-easy/blob/master/docker-compose.yml#L24

**Reply by peteh** [reply]
So how do I set different settings for different clients then? You just linked to the global option which does not fulfill the use case I mentioned.

**Reply by offizium-berndstorath** [reply]
The feature is not yet implemented tho noted here #1179

**Reply by peteh** [reply]
> The feature is not yet implemented tho noted here #1179

Is it meant per client? As it's written there it just means that the setting currently set for all clients would be configurable from UI instead of env variable.

**Reply by kaaax0815** [reply]
> So how do I set different settings for different clients then?

Its not clearly stated, but yes its per client.

**Reply by peteh** [reply]
> > So how do I set different settings for different clients then?
> 
> Its not clearly stated, but yes its per client.

Perfect. Then I'll just wait until it's available. Thank you very much.
